x/subscription/spec/01_concepts.md

```diff
@@ -23,4 +23,20 @@ Intervals are specified using [crontab syntax](https://crontab.guru/), for examp
 
 Convenient shortcuts:
 
-- `@monthly`: `0 0 * * *`
```

Comment:
the old short was for daily rather than monthly ?

Reply:
yeah, the old one is wrong😂